src/styles/topbar.styl

```diff
@@ -20,3 +20,19 @@
 
         &.hidden-mobile
             display none
+
+.fil-tab-item
+    margin-left 0 !important
+
+    &.fil-tab-item-selected
+        background-color transparent !important
+
+        .fil-tab-icon
+            path
+                fill var(--primaryColor)
+
+    .fil-tab-icon
+        margin-right .5rem
+
+        path
+            fill: var(--shadow7)
```

Comment:
When we can, we now try to use utilitary classes.
 
 So for example : 
-  `.u-ml-0` to set margin-left to 0
- maybe your `margin-right .5rem` can be replaced by `u-mr-1` or `u-mr-2`

About icon color, you can pass the color directly as a prop. Example : `<Icon icon={MyIcon} width={16} height={16} color="var(--primaryColor)"/>`

Reply:
The color prop does not apply the color to correct element

Reply:
Can you send a picture of what is expected (Figma) and what is happening ? Maybe there is an issue with the icon or cozy-ui. But let's not wait this to merge.